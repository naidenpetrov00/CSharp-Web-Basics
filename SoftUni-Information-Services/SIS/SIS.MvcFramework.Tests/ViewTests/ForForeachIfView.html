<h1>Hello!</h1>
@for(i = 1, i <= 7, i++)
{
    if(i == 6 || i == 7)
    {
<p>It's Weekend!!</p>
    }
}