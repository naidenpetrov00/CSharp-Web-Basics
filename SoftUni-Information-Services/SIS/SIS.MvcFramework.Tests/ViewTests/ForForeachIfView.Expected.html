<h1>Hello!</h1>
<p>It's Weekend!!</p>
<p>It's Weekend!!</p>